starboard/shared/starboard/player/filter/filter_based_player_worker_handler.cc

```diff
@@ -126,11 +129,10 @@ bool FilterBasedPlayerWorkerHandler::Init(
       SB_LOG(ERROR) << "Audio channels requested " << required_audio_channels
                     << ", but currently supported less than or equal to "
                     << supported_audio_channels;
-      OnError(
-          kSbPlayerErrorCapabilityChanged,
+      std::string error_message =
           FormatString("Required channel %d is greater than maximum channel %d",
-                       required_audio_channels, supported_audio_channels));
-      return false;
+                       required_audio_channels, supported_audio_channels);
+      return {false, std::move(error_message.c_str())};
```

Comment:
`std::move(error_message.c_str()` doesn't move the content of `error_message`, just want to double check if this is expected.